### PR TITLE
Rename velocity to "Motion Vector"

### PR DESCRIPTION
**Pull request by FrancescoC-unity**
### Purpose of this PR
Rename velocity into motion vectors in HDRP. 

---
### Testing status
**Katana Tests**: https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=hdrp%2Frename-velocity-to-motion-vector&automation-tools_branch=master&unity_branch=trunk [Running]

---
### Overall Product Risks
**Technical Risk**: Low

**Halo Effect**:  Medium

### CODE REVIEW

**Comment by sebastienlagarde** [comment]
@RemyUnity could you make a pass on this PR ? thanks

**Comment by sebastienlagarde** [comment]
@FrancescoC-unity need a new merge, also looks like various test are failing and we still need a test from @RemyUnity

File: com.unity.render-pipelines.high-definition/CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added depth offset input in shader graph master nodes
 - Added a Parallax Occlusion Mapping node
 - Added SMAA support
+<<<<<<< HEAD
```
Comment (FrancescoC-unity) [inline com.unity.render-pipelines.high-definition/CHANGELOG.md]: This didn't merge correctly, will fix.

**Comment by sebastienlagarde** [comment]
@RemyUnity have you done a manual test of this PR ? This is the risk. It is just a renaming, but want to be sure motion vectors still work. 
@FrancescoC-unity will need a new merge from master, sorry :)

**Comment by FrancescoC-unity** [comment]
Done the merge!

**Comment by sebastienlagarde** [comment]
Katana is green except one test failing on master. Merging

**Pull request merged**